Backend/app/users/management/commands/create_sample_data.py

```
@@ -0,0 +1,154 @@
+                self.style.SUCCESS(f'Created therapist: {therapist_user.email}')
+            )
+            self.stdout.write(f'  - PIN: {therapist_profile.therapist_pin}')
+            self.stdout.write(f'  - Pairing Code: {therapist_profile.pairing_code}')
+
+        # Create patients
+        for i in range(options['patients']):
+            patient_user = User.objects.create_user(
+                username=f'patient{i+1}@example.com',
+                email=f'patient{i+1}@example.com',
+                password='password123',
+                first_name=f'Patient',
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)